packages/env/lib/commands/start.js

```diff
@@ -78,13 +84,39 @@ module.exports = async function start( { spinner, debug, update } ) {
 	 */
 	if ( shouldConfigureWp ) {
 		await stop( { spinner, debug } );
+		// Update the images before starting the services again.
+		spinner.text = 'Updating docker images.';
+
+		const directoryHash = path.basename( workDirectoryPath );
+
+		// Note: when the base docker image is updated, we want that operation to
+		// also update WordPress. Since we store wordpress/tests-wordpress files
```

Comment:
>  Since we store wordpress/tests-wordpress files as docker volumes, simply updating the image will not change those files

Oh that's interesting, and makes total sense when I think more about it. This is a very helpful comment 👍

Reply:
No, the database is stored in a separate volume, so no data should be wiped!

I'm not entirely sure why we have volumes for the wordpress directories. I think it might be performance? @noisysocks would know more